frontend/src/components/sidebar/SideBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Drawer from "react-modern-drawer";
+import "react-modern-drawer/dist/index.css";
+import { NavLink } from "react-router-dom";
+import classes from "./SideBar.module.scss";
+import { ReactComponent as SettingsIcon } from "../../assets/icons/settings-circle.svg";
+import { ReactComponent as ConnectIcon } from "../../assets/icons/connect-circle .svg";
+import { ReactComponent as SwitchIcon } from "../../assets/icons/switch.svg";
+import { ReactComponent as PlusIcon } from "../../assets/icons/plus.svg";
+import IconButton from "components/icon-button/IconButton";
+
+export type Sidebar = {
+  toggled: boolean;
+  toggleSidebar: () => void;
+};
+
+function SideBar({ toggled, toggleSidebar }: Sidebar) {
+  // TODO(sideBar-component): Check if it is possible (with currently used library "react-modern-drawer")
+  // to position sidebar bellow navbar instead of overlaping it
+  return (
+    <Drawer
+      open={toggled}
+      direction={"right"}
+      onClose={toggleSidebar}
+      className={classes.sidebar}
+      size={300}
+    >
+      <div className={classes.menu}>
+        <div>
+          <NavLink to={"#"} className={classes.menuItem}>
+            <SwitchIcon className={classes.icon} />
+            <div className={classes.text}>Switch project</div>
+          </NavLink>
+
+          <NavLink to={"#"} className={classes.menuItem}>
+            <ConnectIcon className={classes.icon} />
+            <div className={classes.text}>Connect</div>
+          </NavLink>
+
+          <NavLink to={"#"} className={classes.menuItem}>
+            <SettingsIcon className={classes.icon} />
+            <div className={classes.text}>Settings</div>
+          </NavLink>
+        </div>
+        <div className={`${classes.menuItem} ${classes.footer}`}>
```

Comment:
Nit: I've recently started using the `classnames` npm library to make class manipulation easier/more readable: https://github.com/JedWatson/classnames

This could be replaced with `classNames(classes.menuItem, classes.footer)`, which is a bit more readable.

But this library really shines when you use conditionall logic. It allows you to say:
```jsx
className={classNames({ [classes.foo]: shouldIncludeFooClass, ...otherClasses })}
```
... instead of:
```jsx
className={`${shouldIncludeFooClass ? classes.foo : ''} ${...otherClasses}`}
```